setup.cfg

```diff
@@ -7,7 +7,8 @@ filterwarnings =
     ignore::PendingDeprecationWarning
     ignore::FutureWarning
     ignore::UserWarning
-    ignore::keras.saving.legacy.serialization.CustomMaskWarning
+    # Ignore a spurious warning on tf-nightly related to save model changes.
+    ignore:Custom mask layers require a config
```

Comment:
this filter warning syntax is a bit strange, describer here
https://docs.python.org/3/library/warnings.html#describing-warning-filters